src/gui/VerificationValidationWidget.cpp

```diff
@@ -7,6 +7,15 @@ using Parser = VerificationValidation::Parser;
 
 #define SHOW_ERROR_POPUP true
 
+TestItem::TestItem(QListWidgetItem* _testWidgetItem, int _id, QString _testName, QString _testCommand, bool _hasValArgs, QString _category){

```

Comment:
Move these arguments to the Test class in VerificationValidation.h

Reply:
moving this to the .h resulted in an error im not sure the cause of so i left it were it was